presto-kinesis/src/main/java/io/prestosql/plugin/kinesis/KinesisConnectorConfig.java

```diff
@@ -0,0 +1,348 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kinesis;
+
+import io.airlift.configuration.Config;
+import io.airlift.units.Duration;
+
+import javax.validation.constraints.NotNull;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This Class handles all the configuration settings that is stored in /etc/catalog/kinesis.properties file
```

Comment:
This comment is not accurate since the catalog property file can be named anything and the path might not be under `/etc`

Reply:
corrected